app/public/map.js

```diff
@@ -98,3 +98,106 @@ var nazarethNuggets = (function () { // eslint-disable-line
+  DropDownCategory.setAttribute('name', 'DropDownCategoryName')
+
+  var option1 = document.createElement('option') // input element, text
+  option1.textContent = 'Food'
+  DropDownCategory.appendChild(option1)
+  var option2 = document.createElement('option') // input element, text
+  option2.textContent = 'Info'
+  DropDownCategory.appendChild(option2)
+  var option3 = document.createElement('option') // input element, text
+  option3.textContent = 'Fun Fact!'
+  DropDownCategory.appendChild(option3)
+  var option4 = document.createElement('option') // input element, text
+  option4.textContent = 'View'
+  DropDownCategory.appendChild(option4)
+
+  ff1.appendChild(paraCategory2)
+  ff1.appendChild(DropDownCategory)
+  document.getElementsByClassName('slide-up-tab-content')[0].appendChild(ff1)
+
+  var paraUpload3 = document.createElement('P')
+  var paraUpload3Attr = document.createTextNode('Upload Image: ')
+  paraUpload3.appendChild(paraUpload3Attr)
+
+  var uploadBtn1 = document.createElement('input')  // input element, text
+  uploadBtn1.setAttribute('type', 'file')
+  uploadBtn1.setAttribute('name', 'UploadBtnName')
+
+  ff1.appendChild(paraUpload3)
+  ff1.appendChild(uploadBtn1)
+
+  document.getElementsByClassName('slide-up-tab-content')[0].appendChild(ff1)
+
+  var paraDescription4 = document.createElement('P')
+  var paraDescription4Attr = document.createTextNode('Description: ')
+  paraDescription4.appendChild(paraDescription4Attr)
+
+  var txtBoxDescription1 = document.createElement('textarea')
+  var t = document.createTextNode('')
+  txtBoxDescription1.appendChild(t)
+
+  ff1.appendChild(paraDescription4)
+  ff1.appendChild(txtBoxDescription1)
+
+  document.getElementsByClassName('slide-up-tab-content')[0].appendChild(ff1)
+
+  var paraName = document.createElement('P')
+  var paraNameAttr = document.createTextNode('Your name: ')
+  paraName.appendChild(paraNameAttr)
+
+  var txtboxName = document.createElement('input')
+  txtboxName.setAttribute('type', 'text')
+
+  ff1.appendChild(paraName)
+  ff1.appendChild(txtboxName)
+
+  document.getElementsByClassName('slide-up-tab-content')[0].appendChild(ff1)
+
+  var timesCircleButton = document.createElement('i')
+  timesCircleButton.setAttribute('class', 'fa fa-times-circle fa-3x location-select-cross')
+  timesCircleButton.setAttribute('aria-hidden', 'true')
+  ff1.appendChild(timesCircleButton)
+  document.getElementsByClassName('slide-up-tab-content')[0].appendChild(ff1)
+
+  var checkCircleButton = document.createElement('i')
+  checkCircleButton.setAttribute('class', '  fa fa-check-circle fa-5x location-select-tick')
+  checkCircleButton.setAttribute('aria-hidden', 'true')
+  ff1.appendChild(checkCircleButton)
+  document.getElementsByClassName('slide-up-tab-content')[0].appendChild(ff1)
+
+  var newDiv = document.createElement('div')
+  newDiv.classList.add('slide-up-tab')
+  newDiv.classList.add('nugget-add-container')
```

Comment:
I would suggest a better name for this, e.g. add-form-tab or add-nugget-tab or add-nugget-form